roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -89,6 +89,7 @@
     --ignore-preflight-errors=all
     --allow-experimental-upgrades
     --allow-release-candidate-upgrades
+    --force
```

Comment:
?

Reply:
Looks it got inserted but I forgot to remove, anyway it is safe to have this line.
In past two tries to upgrade kubeadm, it failed without --force and asked to add --force flag (tried from v1.10.3 to v1.10.4 and from v1.10.4 to v1.10.5